cloud/deploys/dev_azure/bin/terraform

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env bash
+
+source cloud/deploys/dev_azure/bin/lib.sh
```

Comment:
what's the goal with this script?

Reply:
It sources your local config from dev_azure, then calls the main script in cloud/shared/bin (the same way that setup and deploy do it)